.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git*
+log/*
+tmp/*
+docs/*
```

Comment:
What's this file for?

Reply:
Oh, this was a leftover of some old experiments on speeding up build times.  It stops things from being loaded into the docker container that is not needed for building.  I'll remove it next commit.

Reply:
Future PR material, I hope? ;)